src/acesrender.cpp

```diff
@@ -144,7 +144,7 @@ int AcesRender::readCameraSenPath( const char * cameraSenPath,
 
             for ( vector<string>::iterator file = cFiles.begin( ); file != cFiles.end( ); ++file ) {
                 string fn( *file );
-                
+                if ( fn.find(".json") == std::string::npos ) continue;
```

Comment:
Just a thought, but is it better to only read files with a .json extension into the cFiles variable instead of testing each file in cFiles to see if it's a .json?